src/gui/header_bar.rs

```diff
@@ -95,6 +95,7 @@ pub mod imp {
                 let about: AboutWindow = builder
                     .object("about")
                     .expect("about.ui to have at least one object about");
+                about.add_link("Donate", "https://www.tubefeeder.de/donate.html");
```

Comment:
Is there a way I can make "Donate" translatable?

Reply:
@Schmiddiii pinging you in case you missed it (I commented and you approved at the same time).

Reply:
I missed it indeed. This should do it:


```suggestion
                about.add_link(&gettextrs::gettext("Donate"), "https://www.tubefeeder.de/donate.html");
```